lib/ansible/modules/storage/netapp/na_ontap_net_routes.py

```diff
@@ -44,28 +44,31 @@
     description:
     - Specify the route metric.
     - If this field is not provided the default will be set to 20.
-  new_destination:
+  from_destination:
```

Comment:
Typically, you shouldn't just remove and replace an option with a new one, as this will break existing playbooks.

Typically here, you would use aliases, to allow it to be addressed by multiple names.

And maybe add deprecations before removing the old one.

Additionally, due to this, `version_added` likely shouldn't be touched.

Reply:
This was actually a bug to use new_destination, as it prevents idempotency.
In this case, it's simpler to break existing playbooks, as they are already broken if they were using new_destination.

Reply:
per @sivel 's request and as @lonico pointed out it is a bug fix, as the change is coming, could we get this change documented that the change occurred for future users just in case they happen to come across it? ex: was `new_destanation` now use `from_destination`. 

Doesn't have to be big, more just showing that it has changed.

Reply:
@carchi8py were you able to document this?

Reply:
I had a question if this should be documented in the module, or somewhere else?

Reply:
It's a change in semantics.
Before we were using name/new_name for rename operations, and name for everything else.
Now, name always indicates the end result, and from_name is used for rename operation.  This solves idempotency issues, and makes the module more declarative.

Reply:
@carchi8py if I didn't echo this on a call, just documenting this in specs, just saying something like _old_ -> _new_ way of calling it.